impl/src/main/java/dao/AnswerDao.java

```diff
@@ -29,7 +28,7 @@
 
     @Query(
         SELECT_ANSWER +
-            "\"user\".\"name\" AS created_by " +
+            "\"user\".\"name\" AS created_by, \"user\".picture " +
```

Comment:
Är det för att det är reserverade ord som gör att vi måste dubbelfnutta user?
Sen kanske du kunde ta bort dubbelfnuttningen på name här då om det inte behövs. Det verkar ju inte behövas på rad 40 iaf.

Reply:
User is reserved but we could use an alias, e.g. INNER JOIN \"user\" u, to minimise fnutts. :)

Reply:
as well , we should rename the tables to plural. But thats another issue